src/main/java/sirius/biz/protocol/JournalData.java

```diff
@@ -133,12 +133,12 @@ public static void addJournalEntry(BaseEntity<?> entity, String changes) {
     /**
      * Adds an entry to the journal of the given entity.
      *
-     * @param targetId   the id of the entity under which the entity will be written
      * @param targetType the type of the entity under which the entity will be written
+     * @param targetId   the id of the entity under which the entity will be written
      * @param targetName the name identifying the entity, which can differ from the owner entity
      * @param changes    the entry to add to the journal
      */
-    public static void addJournalEntry(String targetId, String targetType, String targetName, String changes) {
+    public static void addJournalEntry(String targetType, String targetId, String targetName, String changes) {
```

Comment:
This is a breaking change, isnt it?

Reply:
No, because this method is new within this PR.